dlib/simd/simd4f.h

```diff
@@ -436,12 +524,15 @@ namespace dlib
 
 // ----------------------------------------------------------------------------------------
 
-    inline simd4f reciprocal_sqrt (const simd4f& item) 
-    { 
+    inline simd4f reciprocal_sqrt (const simd4f& item)
+    {
 #ifdef DLIB_HAVE_SSE2
-        return _mm_rsqrt_ps(item); 
+        return _mm_rsqrt_ps(item);
 #elif defined(DLIB_HAVE_VSX)
-        return vec_rsqrt(item());
```

Comment:
Is this what you meant to put here?  I would imagine that rather than replacing this line you meant to add `#elif defined(DLIB_HAVE_NEON)`

Reply:
On raspberry pi 3b+, it works!